addons/website/views/snippets/s_freegrid.xml

```diff
@@ -0,0 +1,40 @@
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_03" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-5 col-lg-3 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 2; grid-area: 1 / 2 / 6 / 5; --grid-item-padding-y: 8px; --grid-item-padding-x: 16px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_13" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-5 col-lg-3 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 3; grid-area: 6 / 2 / 11 / 5; --grid-item-padding-y: 8px; --grid-item-padding-x: 16px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_10" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-8 col-lg-3 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 4; grid-area: 2 / 5 / 10 / 8; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_05" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-10 col-lg-3" style="z-index: 5; grid-area: 1 / 8 / 11 / 11; --grid-item-padding-y: 8px; --grid-item-padding-x: 16px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_14" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-2 g-height-8 col-lg-2 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 6; grid-area: 2 / 11 / 10 / 13; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_16" alt=""/>
+                </div>
+                <div class="o_grid_item g-height-5 g-col-lg-7 col-lg-7" style="z-index: 7; grid-area: 12 / 2 / 17 / 9; --grid-item-padding-y: 16px; --grid-item-padding-x: 16px;">
+                    <h2 style="text-align: left;">A deep dive into what makes our products innovative</h2>
+                    <p class="lead" style="text-align: left;">Write one or two paragraphs describing your product or services. To be successful your content needs to be useful to your readers. Start with the customer – find out what they want and give it to them.</p>
+                </div>
+                <div class="o_grid_item g-height-2 g-col-lg-3 col-lg-3" style="z-index: 8; grid-area: 12 / 9 / 14 / 12; --grid-item-padding-y: 16px; --grid-item-padding-x: 16px;">
+                    <p style="text-align: right;">
+                        <a t-att-href="cta_btn_href" class="btn btn-primary btn-lg"><t t-out="cta_btn_text">Learn more</t></a>
```

Comment:
what is the propose of `<t t-out="cta_btn_text">`? Is it useful for anything ?

Reply:
Thanks for your review/comment @bvr-odoo 
Honestly I've seen that on somes snippets (s_cover, s_call_to_actions,...) that had buttons, so I added it.. But maybe it isn't useful in this snippet..

Reply:
> Thanks for your review/comment @bvr-odoo Honestly I've seen that on somes snippets (s_cover, s_call_to_actions,...) that had buttons, so I added it.. But maybe it isn't useful in this snippet.. Do you think I should remove it?

I checked in the code, and the purpose is to allow the text to be replaced by a theme based on what the user selected as the main purpose of their website in the configurator (e.g., if they selected "schedule appointments," the button text will be replaced with "Schedule an appointment").

However, in your case, the text is already being replaced in themes using XPath, so I don't think it's necessary. You can remove it.

Ping me once the changes are made here and in the "design-themes" PR (+ rebase your branch ;) ) Thanks.

Reply:
```suggestion
                        <a href="#" class="btn btn-primary btn-lg">Learn more</a>
```
`t-att-href="cta_btn_href" `must also be replaced

Reply:
Thanks for the explanation!

Updated & rebased both branches :)

Reply:
> Thanks for the explanation!
> 
> Updated & rebased both branches :)

You forgot to push the changes ;)

Reply:
![image](https://github.com/user-attachments/assets/333c74e0-3c05-4103-8c6c-62ae9f7b0608)
damn -_- my bad pushed again.